.vscode/extensions.json

```diff
@@ -1,7 +1,11 @@
 {
   "recommendations": [
+    "EditorConfig.EditorConfig",
+    "esbenp.prettier-vscode",
```

Comment:
This is actually not recommended because we use Prettier via ESLint to handle all the formatting:

![image](https://user-images.githubusercontent.com/44849/108880826-53f1b780-760b-11eb-85b0-f9ba4c4d87cf.png)